src/BitNode/BitNodeUtils.ts

```diff
@@ -9,3 +14,63 @@ export function isBitNodeFinished(): boolean {
   }
   return wd.backdoorInstalled;
 }
+
+export function canAccessBitNodeFeature(bitNode: number): boolean {
+  return Player.bitNodeN === bitNode || Player.activeSourceFileLvl(bitNode) > 0;
+}
+
+export function knowAboutBitverse(): boolean {
+  for (const [, sfActiveLevel] of Player.activeSourceFiles) {
```

Comment:
Just a style thing, but when only iterating one side of the map I would recommend using .keys()/.values() instead of using a partial array
```suggestion
  for (const sfActiveLevel of Player.activeSourceFiles.values()) {
```